architecture/additional_concepts/storage.adoc

```diff
@@ -405,6 +405,70 @@ cluster.
 
 The CLI shows the name of the PVC bound to the PV.
 
+[[pv-mount-options]]
+
+=== Mount Options
+[IMPORTANT]
+====
+Mount Options is a Technology Preview feature and it is only available for manually provisioned persistent volumes.
+ifdef::openshift-enterprise[]
+Technology Preview features are not supported with Red Hat production service
+level agreements (SLAs), might not be functionally complete, and Red Hat does
+not recommend to use them for production. These features provide early access to
+upcoming product features, enabling customers to test functionality and provide
+feedback during the development process.
+
+For more information on Red Hat Technology Preview features support scope, see
+https://access.redhat.com/support/offerings/techpreview/.
+endif::[]
+====
+
+You can specify mount options while mounting a persistent volume by using the annotation `volume.beta.kubernetes.io/mount-options`.
+
+For example:
+
+====
+[source,yaml]
+----
+  apiVersion: v1
+  kind: PersistentVolume
+  metadata:
+    name: pv0001
+    annotations:
+      volume.beta.kubernetes.io/mount-options: rw,nfsvers=4,noexec <1>
+  spec:
+    capacity:
+      storage: 1Gi
+    accessModes:
+    - ReadWriteOnce
+    nfs:
+      path: /tmp
+      server: 172.17.0.2
+    persistentVolumeReclaimPolicy: Recycle
+    claimRef:
+      name: claim1
+      namespace: default
```

Comment:
@gaurav-nelson I'm not sure, but will the above being one space out from the edge cause problems?